src/pages/Assembly/index.tsx

```diff
@@ -79,7 +79,12 @@ const AssemblyPage: React.FC = () => {
               Additional annotations produced by workflows run outside the scope
               of MGnify’s versioned pipelines.
             </p>
-            <AssemblyExtraAnnotations />
+            <ExtraAnnotations
+              annotationsPageProp="assembly-annotations-page"
+              annotationsPageSizeProp="assembly-annotations-page_size"
+              namespace="assemblies"
+              entityName="assembly"
```

Comment:
c.f. comment on the Run page – maybe the page props are inferable from namespace.